images/Makefile.common.in

```diff
@@ -13,7 +13,7 @@ IMAGE?=$(REGISTRY)/$(IMAGE_NAME):$(TAG)
 export DOCKER_CLI_EXPERIMENTAL=enabled
 
 # build with buildx
-PLATFORMS?=linux/amd64,linux/arm64,linux/ppc64le
```

Comment:
I think we should probably stop building the binary for platforms that cannot run an image we've prebuilt as well. We are wasting unreasonable time on broken or flaky builds for platforms that have roughly N=0 users.

Reply:
In this case qemu coredumps now and hangs the image build indefinitely. I'm out of time to waste debugging this.

Reply:
+1000